### ISSUE TITLE
@swc/plugin-emotion Makes wrong params when "styled" has callable options

### ISSUE THREAD

**Issue by ImBrek** [swc-project/plugins#201]
```
const ContainerWithOptions = styled('div', makeOptions())`
  color: hotpink;
`
```

transforms to

```
const ContainerWithOptions = /*#__PURE__*/ styled('div', makeOptions(), {
    target: "e1imlkhz0",
    label: "ContainerWithOptions"
})("color:hotpink");
```

I think it is wrong. Third param should extend result of `makeOptions()`

**Comment by Codex-** [comment]
Did you still want this issue open since you fixed this with #202 ?

**Comment by ImBrek** [comment]
No no, everything is ok
